### PR TITLE
Update colormanagement.xml

### LINKED ISSUE #4279: [user manual] section on working color space needs updating
Looking through the current state of the user manual for areas which still need updating, I noticed section [3.2.6.4, "darktable's color spaces"](https://darktable.gitlab.io/doc/en/color_management.html#darktable_color_spaces) is still out of date, but I'm not comfortable re-writing it myself. If somebody with a better understanding of the current situation can correct the basic information, I would be happy to copy edit the results.

### PR DESCRIPTION
**Pull request by junkyardsparkle**
Most changes are to reflect that the working space is no longer CIELAB, but also add and update some external links, and compress a couple of paragraphs. Also, I assume sections of preferences dialog should no longer be called "tabs"... is that correct? Closes #4279

### CODE REVIEW

File: doc/usermanual/darkroom/concepts/colormanagement.xml [hunk 1]
```diff
@@ -24,15 +24,16 @@
...
         configured) for an accurate color rendition on screen. Multi-screen setups are fully
-        supported as long as a system service like <emphasis>colord</emphasis> is in place and
-        properly set up to inform darktable about the correct monitor profile.
+        supported as long as a system service like
+        <ulink url="https://www.freedesktop.org/software/colord/intro.html"><emphasis>colord</emphasis></ulink>
+        is in place and properly set up to inform darktable about the correct monitor profile.
       </para></listitem>
 
       <listitem><para>
         Output files can be encoded in one of darktable's built-in profiles, like
```
Comment (TurboGit) [inline doc/usermanual/darkroom/concepts/colormanagement.xml:33]: In output profile I would also list: ProPhoto, Rec2020, Rec709. Maybe not PQ P3 RGB, HLG P3 RGB, which are not main stream.

**Review by TurboGit — COMMENTED** [review]
Minor comment about output profile.

File: doc/usermanual/darkroom/concepts/colormanagement.xml (hunk 1 above)
Comment (junkyardsparkle) [reply doc/usermanual/darkroom/concepts/colormanagement.xml]: Done. Also another wikipedia link and minor copy edits.

**Comment by TurboGit** [comment]
Thanks !

**Comment by TurboGit** [comment]
About the "tabs", I think it can be called tabs, it was horizontal tabs, it is now vertical. But section is also ok and maybe more friendly.

**Pull request merged**